src/cli.ts

```diff
@@ -52,15 +52,23 @@ export interface OdoChannel {
     print(text: string): void;
 }
 
+export const openShiftOutput = {
```

Comment:
No need to create new object with map. There is always one OpenShift output channel. Just add this method to Cli:

``` JavaScript
showOutputChannel() {
   this.odoChannel.show();
}